pinc/special_colors.inc

```diff
@@ -154,9 +154,10 @@ function echo_special_legend($projects_where_clause)
     $legend_text = _("Legend for Special Day Colors");
 
     $currspecs_result = DPDatabase::query("
-            SELECT distinct special_code as spec FROM projects
-            WHERE $projects_where_clause
-        ");
+        SELECT distinct special_code as spec
+        FROM projects
+        WHERE $projects_where_clause
+    ");
```

Comment:
I consider this a similar case to the last one. Because this doesn't have the additional `sprintf()` it doesn't suffer from the weird/additional indenting, which is why I broke those few out into `$sql =; DPDatabase::query($sql);`

Reply:
Would it then be more consistent if the second case in the initial description (`sprintf()` with multi-line SQL) had the open quotes on the same line as the `sprintf(`, like this...
```
$sql = sprintf("
    SELECT *
    FROM table
    WHERE x = %d
    ",
    $some_variable
);
```
None of the other cases have the open quote on its own - it stays with whatever precedes it.
Just a question - I'm happy either way.

Reply:
It appears php-cs-fixer would change it to
```
$sql = sprintf(
    "
    SELECT *
    FROM table
    WHERE x = %d
    ",
    $some_variable
);
```

Reply:
Right, php-cs-fixer wants each arg to a multi-line call each on a separate line.

Reply:
Ah, I see. It's when there are multiple args. Thanks for explaining

Reply:
Yes, thanks for explaining. I was mostly confused by the difference in where the `$results` assignment was, strictly on the basis of standardization (remember, when one is learning, it's not always easy to know what's best to use when there are more ways than one to do something!).